db/migrate/20220321234229_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
```

Comment:
- [ ] Well done so far building the migrations of your app and adding the references correctly. However, its seems that you might have missed adding the `posts_counter` column, so let's do that. :pray: :smile:

Reply:
thanks 😸